shell/platform/linux/fl_standard_message_codec_test.cc

```diff
@@ -0,0 +1,989 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/linux/public/flutter_linux/fl_standard_message_codec.h"
+#include "flutter/shell/platform/linux/testing/fl_test.h"
+#include "gtest/gtest.h"
+
+// FIXME(robert-ancell) Have two cases where endianess matters
```

Comment:
I don't think we build for any machines with different endianness. (I ignored it in the C++ implementation.)

Reply:
OK, I'll update this to a note.